### DISCUSSION TITLE
Contracts deployment

### DISCUSSION THREAD

**Discussion by rndquu** [pavlovcik/uad-ui#654] [Q&A]
Questions regarding ubiquity contracts deployment

This is a an issue for the deployment script: https://github.com/ubiquity/ubiquity-dollar/issues/406

As far as I understand we implemented only the `development` migration script

1. This https://github.com/ubiquity/ubiquity-dollar/blob/development/packages/contracts/package.json#L31 seems to be the old deploy script (used before foundry was integrated). Can we delete it?
2. What is the difference between these 2 deployment scripts: https://github.com/ubiquity/ubiquity-dollar/blob/development/packages/contracts/package.json#L32C1-L33 ? They seem to be doing the same job. Perhaps in order to maintain a consistency we should only use forge solidity scripts for development, staging and production deployments
3. We moved to the diamond contract implementation. Does anybody know what exact data (perhaps user balances, or staked amounts) should be migrated to new contracts? I mean we are basically going to deploy contracts from 0 so perhaps there is something that users want to be migrated as well.

**Comment by 0x4007** [comment]
> 1. 

@zgorizzo69 can confirm

> 2.

I'm assuming that one was for only deploying the Diamond contracts.

> 3. 

I assume balances only

@zgorizzo69 these are mostly questions for you.
